source/documentation/gluster-hyperconverged/chap-Single_node_hyperconverged.html.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Deploying oVirt and Gluster Single node Hyperconverged
+---
+
+# Chapter 6: Deploying a Single Node oVirt and Gluster Hyperconverged
+
+## Pre-requisites
+
+* A single host with  Enterprise Linux 7 or oVirt Node. Refer [Enterprise Linux Hosts](../install-guide/chap-Enterprise_Linux_Hosts) or [oVirt Nodes](../install-guide/chap-oVirt_Nodes)
+
+* You must have at least 2 interfaces on the host, so that the frontend and backend traffic can be separated out. Having only one network will cause the engine monitoring, client traffic, gluster I/O traffic to all run together and interfere each other. To segregate the backend network, the gluster cluster is formed using the backend network addresses, and the nodes are added to the engine using the frontend network address.
+
+* You must have a fully qualified domain name prepared for your Engine and the host. Forward and reverse lookup records must both be set in the DNS. **The Engine should use the same subnet as the management network.**
+
+* You must have configured passwordless ssh between the host to itself. gdeploy uses Ansible playbooks and the ability to remotely execute commands is a pre-requisite.
```

Comment:
how?

Reply:
I can add the steps to configure this and refer here